tests/unit/export_case_automated/test.md

```diff
@@ -0,0 +1,55 @@
+# Setup
+1. Install tmt
+```bash
+sudo dnf install -y tmt-all
+```
+2. Create and use a brand new git repo
+```bash
+cd $(mktemp -d)
+git init
+```
+# Test
+## Step
+Verify tmt shows help page
+```bash
+tmt --help
+```
+## Expect
+Text similar to the one bellow is displayed
+```
+Usage: tmt [OPTIONS] COMMAND [ARGS]...
+
+  Test Management Tool
+
+Options:
+...
+```
+## Step
+Check that error about missing metadata is sane
+```bash
+tmt tests ls
+```
+## Expect
+```
+ERROR  No metadata found in the '.' directory. Use 'tmt init' to get started.
+```
+## Step
+Initialize metadata structure
+```bash
+tmt init
+```
+## Expect
+1. Metadata structure was created
+```bash
+$ cat .fmf/version
+1
+```
+
+### Expect
```

Comment:
According to the [spec](https://tmt.readthedocs.io/en/latest/spec/tests.html#manual) the `Step` and `Expect` sections must be in pairs. This test data uses unsupported combination of two `Expect` sections following one another. I don't think we should/need to support such cases and cover them by tests.

Reply:
I have used the [comment](https://github.com/psss/tmt/pull/641#pullrequestreview-613286106) from Lukas. 
> IMHO it is better to ignore Nitrates' structure than to loose information.
That's why I added this structure... to check 2 different headings (h2 and h3) with the same `Expect`. 
But after your comment (According to the [spec](https://tmt.readthedocs.io/en/latest/spec/tests.html#manual)) I found that it's necessary to check `## Test Step`, `# Test .*`, `## Result`, `## Expected Result` in `export.py`. 
And update tests with these data.

Reply:
Ok, understood. I guess that @lukaszachy's main point was that it does not matter if we put the manual test instructions into a different (wrong) Nitrate field, just to make sure that the content is there. But for fields which are clearly not aligned with the specification I'm not sure what would be the best to do... @lukaszachy, @hegerj, any recommendation?

Reply:
Fail with an error. However this should be checked by lint.

Reply:
Maybe fail is too much? Just ignore incorrect structure...

Reply:
If you will user know that something was ignored and continue then OK. However ignoring user data without any feedback is IMHO a no-go.

Reply:
Sounds reasonable. If the format does not respect the specification we should raise an error. Agreed this should be task for `tmt test lint`. During export I would probably print a warning, e.g. `Invalid level 3 header 'Expect'`.

Reply:
If I got your point, I think it makes sense to split up these activities. For the current PR I would do:
1. Extend `export.py` to handle `## Test Step, # Test .*, ## Result, ## Expected Result`
2. Update the test (`test.md`) according to the spec.

Create a separate issue:
1. `tmt test lint` should check If the format of test.md respects the specification. Probably we can add a list of allowed headers for export operation and if `test.md` has headers not from the list -> print a warn.
2. Call `tmt test lint` during export of `test.md` (???) and probably print a warning, e.g. `Invalid level 3 header 'Expect'`

Reply:
Yes, agreed.

Reply:
`test export --nitrate .` for [file](https://github.com/psss/tmt/pull/641/files#diff-0ec5ff23c10e2b1611d192aa4c6547909b70b1e04b726b04f8aaeb5092813da4)  can be found in TCMS: /case/609928